src/app/aliases/aliases.component.ts

```diff
@@ -0,0 +1,59 @@
+import { Component, OnInit } from '@angular/core';
+import { AliasesService } from './state/aliases.service';
+import { AliasesQuery } from './state/aliases.query';
+import { ActivatedRoute, Router } from '../test';
+import { Observable } from 'rxjs';
+import { Organization, Collection } from '../shared/swagger';
+import { Base } from '../shared/base';
+import { takeUntil } from 'rxjs/operators';
+
+@Component({
+  selector: 'aliases',
+  templateUrl: './aliases.component.html',
+  styleUrls: ['./aliases.component.scss']
+})
+export class AliasesComponent extends Base implements OnInit {
+
+  loading$: Observable<boolean>;
+  organization$: Observable<Organization>;
+  collection$: Observable<Collection>;
+
+  public type;
+  public alias;
+  // Types contains resource types that support aliases
+  public types = [ 'organizations', 'collections' ];
+  constructor(private aliasesQuery: AliasesQuery,
+              private aliasesService: AliasesService,
+              private route: ActivatedRoute,
+              private router: Router
+  ) {
+    super();
+  }
+
+  ngOnInit() {
+    this.type = this.route.snapshot.paramMap.get('type');
+    this.alias = this.route.snapshot.paramMap.get('alias');
+
+    this.loading$ = this.aliasesQuery.loading$;
+
+    if (this.types.includes(this.type)) {
```

Comment:
I don't think this if is necessary, given the explicit tests you have for `organizations` and `collections` inside this if, but I could be missing something.

Reply:
I think that is a relic of something I was trying out before, I'll remove it.